app/org/sagebionetworks/bridge/validators/SignUpValidator.java

```diff
@@ -22,15 +27,33 @@ public void validate(Object object, Errors errors) {
         SignUp signUp = (SignUp) object;
 
         if (StringUtils.isBlank(signUp.getEmail())) {
-            errors.rejectValue("email", "null or blank");
+            errors.rejectValue("email", "is required");
         } else if (!emailValidator.isValid(signUp.getEmail())){
-            errors.rejectValue("email", "not a valid email address");
+            errors.rejectValue("email", "must be a valid email address");
         }
         if (StringUtils.isBlank(signUp.getUsername())) {
-            errors.rejectValue("username", "null or blank");
+            errors.rejectValue("username", "is required");
         }
         if (StringUtils.isBlank(signUp.getPassword())) {
-            errors.rejectValue("password", "null or blank");
+            errors.rejectValue("password", "is required");
+            return;
+        }
+        // validate password
```

Comment:
Down the road I feel we will need PasswordValidator that takes a PasswordPolicy object and validates against it.